src/views/menus/AccountMenu.tsx

```diff
@@ -108,7 +132,7 @@ export const AccountMenu = () => {
               </WithTooltip>
             </Styled.AddressRow>
             <Styled.AddressRow>
-              {walletType && (
+              {walletType && walletType !== WalletType.Privy && (
```

Comment:
We need this section to show the icon in the dropdown menu:
<img width="283" alt="Screenshot 2024-04-01 at 1 59 23 PM" src="https://github.com/dydxprotocol/v4-web/assets/1062252/b61409ad-c5d4-40ee-84df-4b265ea71923">
without it, it looks like this:
<img width="276" alt="Screenshot 2024-04-01 at 1 59 02 PM" src="https://github.com/dydxprotocol/v4-web/assets/1062252/5f3fcde0-e6cf-4e4f-813f-d0f256fcfc93">

All we need to do is re-use `walletIcon` here

Reply:
ah right, oops